pkg/logsafe/logsafe.go

```diff
@@ -69,6 +73,18 @@ func (c Cmd) AppendLoggable(vl ...string) Cmd {
 	return c
 }
 
+// AppendLoggableBool appends a boolean value in a "value\no-value" format.
+// In case of "true" value it will append "key".
+// In case of "false" value it will append "no-key"
+func (c Cmd) AppendLoggableBool(k string, v bool) Cmd {
+	if v {
+		c = append(c, arg{key: k})
+	} else {
+		c = append(c, arg{key: fmt.Sprintf("%s%s", noboolpref, k[2:])})
```

Comment:
I suggest changing the comment to the function to explicitly note that "--key" is changing to "--no-key". Or even better to add explicit check to the function that we have "--" in the beginning of the key.
Without a context, k[2:] looks confusing here.